static/css/bureau-structure.less

```diff
@@ -54,18 +59,18 @@
             padding-top: 45px;
             border-width: 0;
 
-            &:before, 
+            &:before,
             &:after {
                 content: '';
-                position: absolute; 
-                top: 0; 
-                width: 50%; 
+                position: absolute;
+                top: 0;
+                width: 50%;
                 height: unit(@grid_gutter-width, px);
                 border-color: @gray-50;
                 border-style: solid;
                 border-width: 0;
             }
-            
+
             &:first-child {
                 padding-right: unit(@grid_gutter-width / 2, px);
```

Comment:
[`@grid_gutter-width`](https://github.com/cfpb/cf-grid/blob/gh-pages/src/cf-grid.less#L42) is already in pixels. Looks like divide by 2 doesn't need the `unit()` helper (per https://github.com/cfpb/cfgov-refresh/pull/401/files#diff-6b6af7d41d4b8c5dc14e6baaa8c1cef2R74), but double-check.